### PR TITLE
Add sites, delete trailing spaces

### PR DESCRIPTION
**Pull request by homebysix**
Add:
- Credit Karma
- Her Future
- Storenvy

Modify:
- Dwolla
- Remove trailing spaces

Contains changes originally submitted in https://github.com/justdeleteme/justdelete.me/pull/660.

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -1065,6 +1065,17 @@
     },
 
     {
+        "name": "Credit Karma",
+        "url": "https://help.creditkarma.com/hc/en-us/articles/202466954-How-do-I-cancel-my-membership-",
```
Comment (Nightfirecat) [inline sites.json]: This should be a direct link to deactivate the account: https://www.creditkarma.com/myprofile/security/deactivate

File: sites.json [hunk 2]
```diff
@@ -1478,7 +1489,7 @@
 
     {
         "name": "Dwolla",
-        "url": "http://help.dwolla.com/customer/portal/articles/269223-how-can-i-delete-my-dwolla-account-",
+        "url": "http://help.dwolla.com/customer/en/portal/articles/1203269-closing-your-dwolla-account",
```
Comment (Nightfirecat) [inline sites.json]: Like with Credit Karma, this is better as the direct deactivation link: http://www.dwolla.com/deactivate

**Comment by Nightfirecat** [comment]
Looks good otherwise. Thanks in advance for your contribution!

**Comment by homebysix** [comment]
Good suggestions, thanks!

**Pull request merged**